src/holofuel/components/Header/Header.test.js

```diff
@@ -62,3 +64,43 @@ it('should render lst 6 of agent id when agent is loading', () => {
   const { getByText } = renderWithRouter(<Header {...props} />)
   expect(getByText(presentAgentId(props.agent.id))).toBeInTheDocument()
 })
+
+// TODO: Resolve Provider Context Conflicts >
+it.skip('should copy the agentId when clicked and trigger the proper flash message', async () => {
+  const FlashMessageContext = createContext()
```

Comment:
I would just use `jest.mock` to mock FlashMessage here, rather than set up the whole FlashMessage context.